projects/igniteui-angular-i18n/README.md

```diff
@@ -0,0 +1,10 @@
+# igniteui-angular-i18n
+
+## Purpose
+
+This package stores resource strings for the Ignite UI for Angular `i18n`. The English resource strings and the interfaces for each localizable component's `resourceStrings` property are stored in the [Ignite UI for Angular](https://github.com/IgniteUI/igniteui-angular) repository. The interfaces can be used as a schema for creating custom resource strings.
```

Comment:
>property are stored in the [Ignite UI for Angular](https://github.com/IgniteUI/igniteui-angular) repository
Kind of not accurate anymore, more like in the main igniteui-angular project/pacakge? The link I guess can be the same since this readme is also used in npm.
I think this also desperately needs a link to the localization docs for usage instructions :) And maybe a link to the repo it was moved from at the end, though not really that big of a deal